docs/changelog.rst

```diff
@@ -1,6 +1,13 @@
 Changelog
 ---------
 
+2.9.49.1
+^^^^^^^^
+
+Friday, November 29th, 2024
```

Comment:
Thanks, the only thing I was going to change was this date. Want to apply this or should I take care of it?
```suggestion
Friday Dec 5 2024
```

Reply:
Or better to just have it match the test release?

Reply:
Let's match the test release.